package.json

```diff
@@ -7,14 +7,15 @@
   },
   "scripts": {
     "start": "npm run web",
+    "analyze": "source-map-explorer build/static/js/*.js",
```

Comment:
I think we need some quotation marks if we use a folder...at least it didn't work in my computer until I wrote:
`"analyze": "source-map-explorer 'build/static/js/*.js'",`